crates/datasources/Cargo.toml

```diff
@@ -75,6 +75,7 @@ scylla = { version = "0.12.0" }
 glob = "0.3.1"
 indexmap = "2.2.5"
 async-sqlite = "0.2.2"
+json-stream = { git = "https://github.com/tychoish/json-stream", rev = "bd4990fab95f789740a75a8eea98d5dac1f0160a" }
```

Comment:
What's the rationale? Are there changes to this? Can those be upstreamed?

Reply:
Yeah, this is just the content of a PR that's open: https://github.com/resyncgg/json-stream/pull/5

The maintainer is responsive and said he'd look at it by the end of the week.